scripts/AbinsModules/GeneralLoadAbInitioTester.py

```diff
@@ -73,8 +73,9 @@ def _check_reader_data(self, correct_data=None, data=None, filename=None, extens
         self.assertEqual(correct_data["attributes"]["ab_initio_program"], data["attributes"]["ab_initio_program"])
         # advanced_parameters is stored as a str but we unpack/compare to dict
         # so comparison will tolerate unimportant formatting changes
-        self.assertEqual(correct_data["attributes"]["advanced_parameters"],
-                         json.loads(data["attributes"]["advanced_parameters"]))
+
+        # self.assertEqual(correct_data["attributes"]["advanced_parameters"],
```

Comment:
Good spot! This was to make the tests remain useful while reworking what advanced parameters exist. Uncommenting it will probably break something and then I'll regenerate the appropriate data files.

Reply:
Have pushed up an update to the test files and restoring this.

However, on reflection I think it does make sense to remove it. Advanced parameters are presumably stored with the reference data in case a parameter modifies how this data is loaded. However, that
- does not happen and
- is not currently planned to happen

so this check could be removed to reduce needless maintenance/replacement of files when the _purpose_ of the test (checking the structure and phonon data import) is unaffected.